test/stdlib/DispatchDeprecationWatchOS.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %swift -typecheck -target i386-apple-watchos2.0 -verify -sdk %sdk %s
-// REQUIRES: OS=watchos
+// REQUIRES: CPU=x86_64, OS=watchos
```

Comment:
Documentation is http://llvm.org/docs/TestingGuide.html
…but some of the LLVM branches used by Swift do not yet support the boolean operators.

Comma-separated clauses act as if they were specified independently on separate lines. `REQUIRES: a, b` requires both `a` and `b` to be true.